updated tests

kac320 opened the pull request.

kac320 committed:
Updated tests

Not sure if these are 100% correct but they should work. They do not have the correct imports yet, but that can be easily changed. Meeting with Jessica and/or Javier in the next few days to talk about the best way to test persistent storage.

kac320 committed:
Updated tests

turtlefarmer commented on test/codeu/chat/server/BasicControllerTest.java:
```diff
@@ -127,4 +127,30 @@ public void testTokenizer() {
+    final User user1 = controller.newUser("user");
+    
+    User.follow(user, user1);
+    assertFalse("ERROR: Failed to follow user");
```
assertFalse takes a condition, not an error string.  You need to be asserting that the follow relationship was correctly created.

kac320 committed:
updated tests

not sure whether this fixes the issue. maybe we can talk about it during the meeting.

kac320 commented:
Changed the tests but not sure if I changed them for the better.

turtlefarmer reviewed (approved):
looks good.  :)
